openapi-validation-core/src/main/java/com/getyourguide/openapi/validation/core/exclusions/InternalViolationExclusions.java

```diff
@@ -36,4 +37,9 @@ private boolean falsePositive404(OpenApiViolation violation) {
     private boolean falsePositive400(OpenApiViolation violation) {
         return violation.getDirection() == Direction.REQUEST && violation.getResponseStatus().orElse(0) == 400;
     }
+
+    private boolean falsePositive405(OpenApiViolation violation) {
+        return violation.getResponseStatus().orElse(0) == 405
+            && "validation.request.operation.notAllowed".equals(violation.getRule());
```

Comment:
Should the rule name here go into a constant? Are all rules defined somewhere?

Reply:
They are not defined anywhere in `swagger-request-validator`. I have now extracted them here in the file at least.

Reply:
Will create a const file now as we also use them in other places.